### ISSUE TITLE
How to map category_id to object label in panoptic_seg

### ISSUE THREAD

**Issue by QasimKhan5x** [facebookresearch/Mask2Former#16]
I am running inference using COCO config. How can I get the object label of each class in the way that it is displayed on a visualized output image? Basically looking for some mapping from category_id to label. 
During inference, I got `category_id`s greater than 91 so I thought the standard COCO mapping won't work.

**Comment by bowenc0221** [comment]
What is the "standard COCO mapping"? The official COCO dataset has labels from 1 to 91 and you can find the mapping from the "categories" in the COCO annotation json files.

![image](https://user-images.githubusercontent.com/25308473/146848059-0803ac84-050b-456c-aac8-94474e8bb0ae.png)

**Comment by bowenc0221** [comment]
For panoptic segmentation, there are 133 categories, and the mapping from class index to class name can be found in the `categories` field in the json file as well.

![image](https://user-images.githubusercontent.com/25308473/147322619-5e297c1d-b4ea-482b-ba1d-97a8d827701e.png)

**Comment by QasimKhan5x** [comment]
Thanks, I found the annotations [here](https://github.com/cocodataset/panopticapi/blob/master/panoptic_coco_categories.json). Just to make sure of two things,
1. I noticed the `category_id` of a person in COCO is 1 but the `category_id` assigned to a person in Mask2Former is 0. The same was for cars So, I just have to subtract the `category_id` by 1 to match the COCO annotations, right?
2. The annotations file I linked has a maximum category ID of 200, but for this project, there are 133 in total
